hw03_mukalamusij.py

```diff
@@ -0,0 +1,104 @@
+# TODO Do not edit this file directly! Instead, create a new file called
+# TODO a03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Joyce Mukalamusi
+# Username: mukalamusij
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/16yXRh2zAV79Lqzp0jjNsU5dQf5AVqUKSO7DxrRJjoeo/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle as h
+
+# Set up the screen
+h.bgcolor("skyblue")
+
+# Create a turtle object
+house = h.Turtle()
+
+
+# function to draw square
+
+def draw_square(side_length):
+    for i in range(4):
+        house.forward(side_length)
+        house.left(90)
+
+
+# function to draw triangle
+
+def draw_triangle(base_length):
+    for i in range(3):
+        # house.width(15)
+        house.forward(base_length)
+        house.left(120)
+
+
+# function to draw door
+
+def draw_door():
+    house.penup()
+    house.goto(-20, -100)
+    house.pendown()
+    house.begin_fill()
+    house.color("brown")
+    draw_square(40)
+    house.end_fill()
+    house.penup()
+    house.goto(0, -100)
+    house.pendown()
+
+
+# func to draw the roof
+def draw_roof():
+    house.width(100)
+    house.penup()
+    house.goto(-50, 100)
+    house.pendown()
+    house.begin_fill()
+    house.color("red")
+    draw_triangle(120)
+    house.end_fill()
+    house.penup()
+    house.goto(0, 0)
+    house.pendown()
+
+
+# main function
+def main():
+    house.penup()
+    house.speed(5) # drawing speed
+    house.setpos(-100, -100)
+    house.pensize(7)
+    house.pendown()
+    draw_square(200)  # draw the house base
+
+    draw_door()  # draw the door
+    draw_roof()  # draw the roof
+
+    house.hideturtle() # hides turtle crusor
+    h.done()
+    # h.exitonclick()
+
+
+# code for text
+house.penup()
+house.goto(0, -200)  # Replace (0, 0) with the desired coordinates
+
+
+text = "Joyce's little simple house"
+font = ("Arial", 16, "normal")
+color = "blue"
+
+house.color(color)
+house.write(text, font=font, align="center")
```

Comment:
This code is outside of `main()`, which makes it the first code that runs. Put it inside of `main()`, before all the other code, and it should work the same.